### ISSUE TITLE
 feat(lib/runtime): implement remaining ParachainHost runtime calls

### ISSUE THREAD

**Issue by kishansagathiya** [ChainSafe/gossamer#3595]
## Issue summary

- Implement remaining two Parachain Host runtime calls mentioned in https://spec.polkadot.network/#sect-anv-runtime-api

- [ ] [ParachainHost_dmq_contents](https://spec.polkadot.network/#id-parachainhost_dmq_contents)
- [ ] [ParachainHost_inbound_hrmp_channels_contents](https://spec.polkadot.network/#id-parachainhost_inbound_hrmp_channels_contents)

**Comment by kishansagathiya** [comment]
@kanishkatn 
If I remember it correctly, you have implemented these two calls, but couldn't get the tests working. If so, could like that work here, so that it doesn't get worked on again by whoever picks up this issue.

**Comment by kanishkatn** [comment]
Yes, you're right about the tests! There was no data onchain so I wasn't able to add these in. It should be easy enough to use dummy data for tests.

I believe there is no such work.
